firebase_admin/_utils.py

```diff
@@ -33,3 +48,25 @@ def _get_initialized_app(app):
 def get_app_service(app, name, initializer):
     app = _get_initialized_app(app)
     return app._get_service(name, initializer) # pylint: disable=protected-access
+
+def handle_requests_error(error, message=None, status=None):
```

Comment:
Please document this method, especially what `error` and `status` are. It's not clear whether `status` is an http status code or a grpc status code, for example.

Reply:
Done